drivers/sensors/sensor.c

```diff
@@ -98,6 +99,7 @@ static int     sensor_poll(FAR struct file *filep, FAR struct pollfd *fds,
 
 static const struct sensor_info g_sensor_info[] =
 {
+  {1,                                 "custom"},
```

Comment:
what does this "1" signify?

Reply:
1 means the element size is one byte, but it can be any value since the first entry content isn't really used by the code. It is just a placehold to avoid other place reference the entry by (type + 1).

Reply:
maybe change to zero if the value is not significant, as it is a more usual "invalid" value

Reply:
Done